udtube/data/conllu.py

```diff
@@ -182,9 +183,21 @@ def __str__(self) -> str:
             line_buf.append(str(token))
         return "\n".join(line_buf) + "\n"
 
+    @staticmethod
+    def _handle_whitespace_token(token: str) -> str:
+        if re.search(r"^\s$", token, flags=re.MULTILINE):
+            # With BERT, white space "words" do not map back to a token index.
```

Comment:
Is this actually specific to BERT? I wonder if this would be better phrased as "Some tokenizers do not map whitespace tokens..." than "With BERT..."